addons/calendar/models/calendar_attendee.py

```diff
@@ -111,8 +111,9 @@ def _send_mail_to_attendees(self, template_xmlid, force_send=False, ignore_recur
         self._notify_attendees(ics_files, invitation_template, rendering_context, force_send)
 
     def _notify_attendees(self, ics_files, mail_template, rendering_context, force_send):
+        email_achived = self.env['res.partner'].search([('active', '=', False)]).mapped('email')
         for attendee in self:
-            if attendee.email and attendee.partner_id != self.env.user.partner_id:
+            if attendee.email and attendee.partner_id != self.env.user.partner_id and attendee.email not in email_achived:
```

Comment:
How about checking attendee not in archived partner instead of checking email ?

Reply:
@Ngquang Because I just want to check archived email and not send email to that archived contact.